SECURITY_CONTACTS

```diff
@@ -4,12 +4,11 @@
 # to for triaging and handling of incoming issues.
 #
 # The below names agree to abide by the
-# [Embargo Policy](https://github.com/kubernetes/sig-release/blob/master/security-release-process-documentation/security-release-process.md#embargo-policy)
+# [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
 # and will be removed and replaced if they violate that agreement.
 #
 # DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
-# INSTRUCTIONS AT https://kubernetes.io/security/
+# INSTRUCTIONS AT ./SECURITY.md
 
-rhatdan
+haircommander
```

Comment:
I think we're missing @saschagrunert :thinking:

Reply:
Oops, added myself as well.